tests/python/pants_test/init/test_plugin_resolver.py

```diff
@@ -143,7 +141,7 @@ def provide_chroot(existing):
 
     # Default to resolving with whatever we're currently running with.
     interpreter_constraints = (
-        InterpreterConstraints([f"=={interpreter.identity.version_str}"]) if interpreter else None
+        InterpreterConstraints([f"=={python_version}.*"]) if python_version else None
```

Comment:
Note - this does subtly change because we are no longer using Pex to hard-pin the patch version of Python in this constraint; we only pin the minor version. That could potentially be a problem if this behavior is particularly material to the test.

Reply:
Correct, behavior of the test. Before, the interpreter constraints would be targeting a specific patch version rather than a specific minor version.